File: ProjetoPython/Pessoa.py
class Pessoa:

    def __init__(self, nome, time):
        self._nome = nome
        self._time = time

    def get_nome (self):
        return self._nome

    def set_nome (self, nome):
        self._nome = nome

    def get_time (self):
        return self._time

    def set_time (self, time):
        self._time = time

    def apresentacao(self):
        print(self._nome, "é uma pessoa que gosta do time", self._time)

File: ProjetoJava/src/main/java/source/projetojava/Pessoa.java
package source.projetojava;

/**
 *
 * @author alejb
 */
public class Pessoa {

  String nome, time;
  
  public Pessoa() {
        
  }
  
  public Pessoa(String nome, String time) {
      
    this.nome = nome;
    this.time = time;
      
  }

  public void apresentacao () {
      
    System.out.println(this.nome + " é uma pessoa que gosta do time " + this.time + ".");
    
  }
  
  public void setNome(String nome) {
    this.nome = nome;
  }

  public String getNome() {
    return this.nome;
  }

  public void setTime(String time) {
    this.time = time;
  }

  public String getTime() {
    return this.time;
  }
  
}